nucleus/dataset_item_uploader.py

```diff
@@ -127,15 +121,15 @@ def _process_append_requests_local(
             requests.append(request)
 
         progressbar = self._client.tqdm_bar(
-            total=len(requests), desc="Local file batches"
+            total=len(requests),
+            desc=f"Uploading {len(items)} items in {len(requests)} batches",
```

Comment:
This makes sense, but for me the progress bar does not update. It just remains at
<img width="678" alt="Screen Shot 2022-09-23 at 16 48 51" src="https://user-images.githubusercontent.com/5999386/191989940-fc5360bf-601d-435e-ab9b-ac90ab407110.png">

Reply:
(discussed it offline)